crates/compilers/src/compilers/multi.rs

```diff
@@ -161,59 +161,85 @@ impl CompilerSettingsRestrictions for MultiCompilerRestrictions {
 /// Settings for the [MultiCompiler]. Includes settings for both Solc and Vyper compilers.
 #[derive(Clone, Debug, Default, PartialEq, Eq, Serialize, Deserialize)]
 pub struct MultiCompilerSettings {
-    pub solc: SolcSettings,
+    pub solc: SolcMultiCompilerSettings,
```

Comment:
this change requires many changes in foundry is it really needed?

Reply:
hmm, let me try another approach that shouldn't require as much changes then